hack/release.sh

```diff
@@ -60,41 +62,26 @@ if (( PUBLISH_RELEASE )); then
 fi
 
 # Build the release
-
 echo "Copying Build release"
 cp ${REPO_ROOT_DIR}/third_party/config/build/release.yaml ${OUTPUT_YAML}
-echo "---" >> ${OUTPUT_YAML}
 
 echo "Building Knative Serving"
-ko resolve ${KO_FLAGS} -f config/ >> ${OUTPUT_YAML}
 echo "---" >> ${OUTPUT_YAML}
+ko resolve ${KO_FLAGS} -f config/ >> ${OUTPUT_YAML}
+
+echo "Building Observability - Elasticsearch & Kibana"
+ko resolve ${KO_FLAGS} -R -f config/monitoring/100-namespace.yaml \
+    -f third_party/config/monitoring/logging/elasticsearch \
+    -f config/monitoring/logging/elasticsearch > ${OBSERVABILITY_ELK_YAML}
+
+echo "Building Observability - Prometheus & Grafana"
+ko resolve ${KO_FLAGS} -R -f config/monitoring/100-namespace.yaml \
+    -f third_party/config/monitoring/metrics/prometheus \
+    -f config/monitoring/metrics/prometheus > ${OBSERVABILITY_PROMETHEUS_YAML}
 
-echo "Building Monitoring & Logging"
-# Make a copy for the lite version
-cp ${OUTPUT_YAML} ${LITE_YAML}
-# Make a copy for the no monitoring version
-cp ${OUTPUT_YAML} ${NO_MON_YAML}
-# Use ko to concatenate them all together.
-ko resolve ${KO_FLAGS} -R -f config/monitoring/100-common \
-    -f config/monitoring/150-elasticsearch \
-    -f third_party/config/monitoring/common \
-    -f third_party/config/monitoring/elasticsearch \
-    -f config/monitoring/200-common \
-    -f config/monitoring/200-common/100-istio.yaml >> ${OUTPUT_YAML}
-# Use ko to do the same for the lite version.
-ko resolve ${KO_FLAGS} -R -f config/monitoring/100-common \
-    -f third_party/config/monitoring/common/istio \
-    -f third_party/config/monitoring/common/kubernetes/kube-state-metrics \
-    -f third_party/config/monitoring/common/prometheus-operator \
-    -f config/monitoring/150-elasticsearch/100-scaling-configmap.yaml \
-    -f config/monitoring/200-common/100-fluentd.yaml \
-    -f config/monitoring/200-common/100-grafana-dash-knative-efficiency.yaml \
-    -f config/monitoring/200-common/100-grafana-dash-knative.yaml \
-    -f config/monitoring/200-common/100-grafana.yaml \
-    -f config/monitoring/200-common/100-istio.yaml \
-    -f config/monitoring/200-common/200-prometheus-exporter \
-    -f config/monitoring/200-common/300-prometheus \
-    -f config/monitoring/200-common/100-istio.yaml >> ${LITE_YAML}
+echo "Building Observability - Zipkin"
```

Comment:
The dependencies of these isn't super clear.  e.g. I wouldn't know which I can install without the others.